index.html

```diff
@@ -1085,6 +942,188 @@ <h3>Implicit Release of Pointer Capture</h3>
 
         </section>
     </section>
+    <section>
+        <h1><dfn>Coalesced and predicted events</dfn></h1>
+
+        <div class="note">This specification does not define how user agents should coalesce or
+        predict pointer movement data. It only specifies the API for accessing this information.</div>
+
+        <section>
+            <h2>Coalesced events</h2>
+
+            <p>For performance reasons, user agents may choose not to send a move event every time
+            the position of a pointer is updated. Instead, they will coalesce (combine/merge) multiple
+            position changes into a single, averaged move event. While this approach helps in reducing
+            the amount of event handling the user agent must perform, it will naturally reduce the
+            granularity and fidelity when tracking a pointer position, particularly for fast and large
+            movements. Using the <code>getCoalescedEvents</code> method it is possible for applications
```

Comment:
Related: some of the existing links from `getCoalescedEvents` can point to the new section 10 or one of its subsections now.  I spotted two: in `pointermove` and in `pointerrawupdate` sections.